### PR TITLE
Feature/same app from multiple devices

### PR DESCRIPTION
**Pull request by AByzhynar**
Provides the information for  #[2763](https://github.com/smartdevicelink/sdl_core/issues/2763)

Proposal: [Support running the same app from multiple devices at the same time](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0204-same-app-from-multiple-devices.md)
This PR is **ready** for review.

### Summary
Add diagram OnAppRegisteredMultipleDevices; add p.9 to MUST section

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by mrapitis** [comment]
@theresalech Ford approves these changes.

**Comment by ShobhitAd** [comment]
Hello @AByzhynar, the changes in the PR don't seem to be related to the https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0204-same-app-from-multiple-devices.md feature but rather seem to be documentation additions/updates for RPCs which were not modified by this feature. 

Please remove any changes which are unrelated to the proposal from this PR. A separate PR could be created with these changes if they are important updates.

**Comment by LuxoftAKutsan** [comment]
@ShobhitAd sorry for this mismatch. Branch commits fixed

File: docs/VR/ChangeRegistration/index.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ Purpose
...
 #### Parameters
 
-|Name|Type|Mandatory|Additional|
-|:---|:---|:--------|:---------|
-|vrSynonyms|String|false|array: true<br>minsize: 1<br>maxsize: 100<br>maxlength: 40|
-|language|[Common.Language](../../common/enums/#language)|true||
-|appID|Integer|true||
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:---------|
+|vrSynonyms|String|false|array: true<br>minsize: 1<br>maxsize: 100<br>maxlength: 40|Request new VR synonyms registration<br>Defines an additional voice recognition command.<br>Must not interfere with any name of previously registered applications from the same device.|
```
Comment (ShobhitAd) [inline docs/VR/ChangeRegistration/index.md]: The new description 
```
Must not interfere with any name of previously registered applications from the same device.
```
should also be added to the `vrSynonyms` param in the `HMIApplication` struct and the `BasicCommunication.OnAppRegistered` RPC

File: docs/TTS/OnResetTimeout/index.md [hunk 2]
```diff
@@ -1,43 +0,0 @@
-## OnResetTimeout
```
[the remark this answers is not in the document]
Comment (LuxoftAKutsan) [reply docs/TTS/OnResetTimeout/index.md]: Removed redundant

File: docs/TTS/ChangeRegistration/index.md [hunk 3]
```diff
@@ -13,10 +13,12 @@ Purpose
 
 #### Parameters
```
[the remark this answers is not in the document]
Comment (LuxoftAKutsan) [reply docs/TTS/ChangeRegistration/index.md]: Fixed according to HMI_API

File: docs/UI/OnResetTimeout/index.md [hunk 4]
```diff
@@ -1,41 +0,0 @@
-## OnResetTimeout
```
[the remark this answers is not in the document]
Comment (LuxoftAKutsan) [reply docs/UI/OnResetTimeout/index.md]: Removed from commi this changes

File: docs/VR/ChangeRegistration/index.md (hunk 1 above)
Comment (LuxoftAKutsan) [reply docs/VR/ChangeRegistration/index.md]: added description with 10ecd74

**Pull request merged**